ci-operator/config/openshift/driver-toolkit/openshift-driver-toolkit-master.yaml

```diff
@@ -48,21 +48,6 @@ tests:
   commands: cat /etc/driver-toolkit-release.json
   container:
     from: driver-toolkit
-- as: e2e-aws-driver-toolkit
```

Comment:
Can you please elaborate on the differences between this test and the one at https://github.com/openshift/release/blob/e22c38c1cf6b4cb8a314a020fe6de5f132ae5e6a/ci-operator/config/openshift/release/openshift-release-master__nightly-4.9.yaml#L546?

Why are we keeping the one in `master_nightly` rather than this one?

Reply:
The tests are totally equivalent. We are keeping the ones in master_nightly-4.[8,9,10] because these are the jobs that will be made "release informing" ([see PR](https://github.com/openshift/release/pull/26071/files)). It is my understanding that all release informing jobs need to be configured under the openshift/release repository ci-operator configuration.

We want to get rid of the jobs configured under the driver-toolkit repo ci-operator config to avoid unnecessary cloud usage for equivalent tests.

Reply:
That make sense, thanks.